pkg/models/repository_configuration.go

```diff
@@ -73,6 +79,16 @@ func (rc *RepositoryConfiguration) DedupeVersions(tx *gorm.DB) error {
 	return nil
 }
 
+func (rc *RepositoryConfiguration) ReplaceNonNilValues(tx *gorm.DB) error {
```

Comment:
```suggestion
func (rc *RepositoryConfiguration) ReplaceEmptyValues(tx *gorm.DB) error {
```
Empty is slightly more precise